src/components/buttons/buttons.vue

```diff
@@ -44,7 +44,7 @@ export default {
     styleActiveRoute(){
       let color = '';
       if(this.$route.path == '/calculator/'+ this.rpayDuration.name ){
```

Comment:
Use route name instead of path
Also return an empty string if route if condition isnt true

Reply:
done